### DISCUSSION TITLE
Git a load of this check-in 🌟   [New to GitHub Check-in]

### DISCUSSION THREAD

**Discussion by queenofcorgis** [community/community#52441] [New to GitHub]
We’re back for a new month with community shoutouts, new categories, and some great content! I’m excited to catch you up with all that is happening in the world of GitHub 🌏. 

🌟**Community Superstars**: A round of applause for these community citizens 👏 
* @ramazancetinkaya: I absolutely loved their [best practices for project security issue](https://github.com/orgs/community/discussions/51445#discussioncomment-5491070) - it was absolutely foolproof! 
* @Majid-J: For [sharing](https://github.com/orgs/community/discussions/50671) what every new to GitHub user needs: a markdown cheatsheet ✏️. Thanks for making sure the community has access to this. 
* @pie972: Great answer from them explaining [the shades of the contribution graph](https://github.com/orgs/community/discussions/51010) 🟩 - appreciate the time you took to respond. 

⬆️ **Level up**: Resources to take your skills to new heights ☁️
* **Secure Code Game** 🎮: A GitHub Security Lab initiative, providing an in-repo learning experience, where learners secure intentionally vulnerable code in a fun way. [Play the game](https://github.com/skills/secure-code-game). 
* 🆕 **Community Interest Categories** 🧠: Take your skills that much further without even leaving the community, we added the [Programming Help](https://github.com/orgs/community/discussions/categories/programming-help?discussions_q=is%3Aopen+sort%3Adate_created+category%3A%22Programming+Help%22) and [Machine Learning](https://github.com/orgs/community/discussions/categories/machine-learning-and-mlops?discussions_q=is%3Aopen+sort%3Adate_created+category%3A%22Machine+Learning+and+MLOps%22)categories to discuss projects, programming, and more! 

📚 **Rousing Reads**: GitHub-produced goodies 🍭
* **Invite the critics and keep learning**: @blueedgetechno on gaining different perspectives, maintaining a growth mindset, and staying true to yourself in [this article](https://github.com/readme/stories/rohan-gupta) for the ReadMe Project. 
* **How to Enable 2FA for your GitHub Account**: Don’t forget, you have until the end of 2023 to enable two factor authentication on your account, @ladykerr [takes us through how to get it set up](https://dev.to/github/how-to-enable-2fa-for-your-github-account-4o60) 🔐. 
* **How GitHub Docs’ new search works**: GitHub Docs recently changed its site-search to Elasticsearch 🔍. [Read how](https://github.blog/2023-03-09-how-github-docs-new-search-works/) it was implemented.

Be sure to check out the Programming Help and Machine Learning categories and let us know what you think. Until we git together next time 👋

**Comment by pie972** [comment]
Hi @queenofcorgis ! Thank you so much for the shoutout, I'm glad my explanation on the shades of the contribution graph was helpful. I'm always happy to contribute to the community in any way I can!

PS: I want to remind everyone to prioritize the security of their GitHub accounts by enabling two-factor authentication. I'm glad to see that @LadyKerr has provided a helpful article on how to set it up.

**Reply by queenofcorgis** [reply]
We're VERY happy to have you contribute! Thanks for signal boosting the reminder to set up 2FA 🚀
